requirements-dev.txt

```diff
@@ -6,5 +6,6 @@ pre-commit
 pytest
 requests
 sphinx
+sphinx_rtd_theme>=1.0.0
```

Comment:
In the rtd environment, sphinx_rtd_theme is preinstalled to some old version, which causes troubles.